site_reliability_guardian_sample/manifest.yaml

```
@@ -0,0 +1,19 @@
+manifestVersion: 1.0
+projects:
+  - name: srg-config-as-code
+    path: srg-config-as-code
+
+environmentGroups:
+  - name: default
+    environments:
+      - name: DT_ENV_ID
+        url:
```

The url will need to be explictly marked as environment variable, else monaco will assume `DT_ENV_URL` is the value of the URL

```suggestion
        url:
          type: environment
```